config/creating_env/external-worker-vm.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  annotations:
+    description: OSBuild Worker
+    name.os.template.kubevirt.io/rhel8.5: "Red Hat Enterprise Linux 8.0 or higher"
+  labels:
+    app: osbuild-worker
+    workload.template.kubevirt.io/server: "true"
+    os.template.kubevirt.io/rhel8.5: "true"
+  namespace: osbuild
```

Comment:
Should it be run on the same namespace? 
If not - please remove it and add the "-n" flag to the cmd